integration_test/Cargo.toml

```diff
@@ -8,5 +8,5 @@ edition = "2018"
 jsonrpc = { path = "..", features = ["minreq_http"] }
 lazy_static = "1.4.0"
 log = "0.4"
-backtrace = "0.3.50"
+backtrace = "0.2.3"
```

Comment:
In 10972a7846412d323911b382010a657a539b5296

Why did you change this? backtrace 0.2 does not link for me.

Reply:
No idea what I was doing, now removed. Note the recent and minimal files are the same.

Reply:
In a065c9fe4b009dc2079cbb2970a83213b602b1cb:

This is not removed.

Reply:
In d4ed55570290e378aee957fdf006d0ce0bb1749b:

Please remove this.

Reply:
My bad. Done, again.